src/one_dragon_qt/widgets/multi_selection_combo_box.py

```
@@ -0,0 +1,548 @@
+            # 步骤2：确定宽度范围（在 min_width 到 max_width 之间）
+            min_w = parent._min_width if parent._min_width is not None else 200
+            max_w = parent._max_width if parent._max_width is not None else float('inf')
+
+            # 计算需要的宽度：确保在 [min_w, max_w] 范围内
+            needed_width = max(min_w, min(total_width, max_w))
+
+            # 步骤3：在添加标签之前设置正确的宽度（关键步骤）
+            # 这样 FlowLayout 会按照正确的宽度进行布局，避免提前换行
+            parent.setMinimumWidth(int(needed_width))
+
+        # 步骤4：清除 FlowLayout 中的所有现有元素
```

Ruff (0.14.10)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)